targeteditor/src/main/scala/explore/targeteditor/AladinContainer.scala

```diff
@@ -95,68 +155,194 @@ object AladinContainer {
           )
       }
 
+    /**
+      * Recalculate the svg, we keep it on the state for better performance
+      *
+      * @return
+      */
+    def initialSvgState: Callback =
+      aladinRef.get
+        .flatMapCB(_.backend.runOnAladinCB(updateSvgState))
+        .void
+
+    /**
+      * Recalculate svg and store it on state
+      *
+      * @param pixelScale
+      * @return
+      */
+    def updateSvgState(v: JsAladin): CallbackTo[Svg] =
+      $.props.flatMap { p =>
+        CallbackTo
+          .pure(
+            visualization
+              .shapesToSvg(GmosGeometry.shapes(p.options.posAngle),
+                           GmosGeometry.pp,
+                           v.pixelScale,
+                           GmosGeometry.ScaleFactor
+              )
+          )
+          .flatTap(svg => $.modState((s: State) => s.copy(svg = svg.some, fov = v.fov)))
```

Comment:
Is the type ascription required?

Reply:
Yeah, it didn't compile for me otherwise